sdc/tests/tests_perf/test_perf_series.py

```diff
@@ -135,6 +135,8 @@ def _test_case(self, pyfunc, name, total_data_length, input_data=None, data_num=
     TC(name='shape', size=[10 ** 7], call_expr='data.shape', usecase_params='data'),
     TC(name='shift', size=[10 ** 8]),
     TC(name='size', size=[10 ** 7], call_expr='data.size', usecase_params='data'),
+    TC(name='skew', size=[10 ** 8], params='skipna=True'),
+    TC(name='skew', size=[10 ** 8], params='skipna=False', input_data=[test_global_input_data_float64[0]]),
```

Comment:
I don't think using `test_global_input_data_float64[0]` is actually a good idea. It contains max_float values which makes impossible to compare results and may cause some other issues.